scripts/sct_process_segmentation.py

```diff
@@ -239,9 +239,8 @@ def main(args):
         fname_output = extract_centerline(fname_segmentation, remove_temp_files, verbose=param.verbose, algo_fitting=param.algo_fitting, use_phys_coord=use_phys_coord)
         if os.path.abspath(fname_output) != output_folder + fname_output:
             shutil.copy(fname_output, output_folder)
-        # to view results
-        sct.printv('\nDone! To view results, type:', param.verbose)
-        sct.printv('fslview ' + fname_segmentation + ' ' + output_folder + fname_output + ' -l Red &\n', param.verbose, 'info')
+        # display viewer syntax
+        sct.display_viewer([fname_segmentation, output_folder + fname_output], l_colormap=['gray', 'red'], l_opacity=['', '1'])
```

Comment:
This is just a replacement of `display_viewer` but `os.path.join(output_folder, fname_output)` would be doable in the process (yeah, the code is ridden with these `folder + filename`).

Reply:
fixed in 47d16609b5e5e364c294b266b31c600e9dcdd8d6